lib/src/responder/simple/simple_node.dart

```diff
@@ -311,7 +311,7 @@ class SimpleNodeProvider extends NodeProviderImpl
   LocalNode _getNode(String path, {bool allowStubs: false}) {
     if (nodes.containsKey(path)) {
       SimpleNode node = nodes[path];
-      if (allowStubs || node._stub == false) {
+      if (node._stub == false || allowStubs) {
```

Comment:
While you're at it: `node._stub == false` --> `!node._stub`

Reply:
Good point, I didn't even notice that.